commands/builds/create.js

```diff
@@ -8,64 +8,56 @@ let uuid = require('node-uuid');
 let os = require('os');
 let path = require('path');
 let request = require('request');
+let exec = require('child_process').execSync
 
-function uploadCwdToSource(app, cwd, fn) {
+function checkTarInstall(tar) {
+  let tarVersion = exec(tar+" --version").toString()
+
+  if (!tarVersion.match(/GNU tar/)) {
+    cli.warn("Builds can fail if their code is not compressed with GNU tar.")
+    cli.warn("Please install it, or specify the '--tar' option")
+    cli.warn("Detected tar version: "+tarVersion.toString())
+  }
+}
+
+function uploadCwdToSource(app, cwd, tar, fn) {
   let tempFilePath = path.join(os.tmpdir(), uuid.v4() + '.tar.gz');
   let ig = ignore().add(fs.readFileSync('.gitignore').toString());
   let filter = ig.createFilter();
 
+  checkTarInstall(tar)
   app.sources().create({}).then(function(source){
-    let archive = archiver('tar', { gzip: true });
-
-    archive.on('finish', function (err) {
-      if (err) { throw err; }
+    exec(tar+" cz -C "+cwd+" --exclude .git --exclude .gitmodules . > "+tempFilePath)
```

Comment:
GNU tar does seem to have a windows version.
http://gnuwin32.sourceforge.net/packages/gtar.htm